math.js

```diff
@@ -14,9 +14,39 @@ function getAge() {
     return 31;
 }
 
+function getCalvin() {
+    return "Cal";
+}
+
+function getDean() {
+    return "DAD?";
+}
+
+function getChampions() {
+    return "Lakers";
+}
+
+function getDupes() {
+    return "Dupes";
+}
+
+function getDupesOne() {
+    return "Dupe One?";
+}
+
+function getDupesTwo() {
+    return "Dupe Two?";
+}
+
 module.exports = {
     sum: sum,
     multiple: multiple,
     getName: getName,
     getAge: getAge,
+    getCalvin: getCalvin,
+    getDean: getDean,
+    getChampions: getChampions,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/dejaniskra/fake-app/pullRequest?prid=8428048)
```suggestion
    getChampions,
```